### PR TITLE
Selected Client UI Design

### PR DESCRIPTION
**Pull request by vgoyal1996**

### CODE REVIEW

File: src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<h1 mat-dialog-title class="title-center">Select Assessment Year</h1>
+<br>
+<div mat-dialog-content>
+  <form [formGroup]="selectForm
+  " (ngSubmit)="navigateToSelectedClientMaster()">
+    <div class="form-group">
+      <mat-form-field>
+        <mat-label>Assessment Year</mat-label>
+        <mat-select name="assessmentYear" formControlName="selected">
+          <mat-option *ngFor="let year of assessmentYears" [value]="year.value">
```
Comment (priya-bansal) [inline src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.html:10]: what if year is empty ? will that never occur ?

File: src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import { FormControl, FormGroupDirective, NgForm, Validators, FormGroup, FormBuilder } from '@angular/forms';
+
+@Component({
+  selector: 'app-assessment-year-dialog',
+  templateUrl: './assessment-year-dialog.component.html',
+  styleUrls: ['./assessment-year-dialog.component.css']
+})
+export class AssessmentYearDialogComponent implements OnInit {
+  private assessmentYears = [
+    {value: '2019-20', viewValue: '2019-20'},
```
Comment (priya-bansal) [inline src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.ts:16]: shouldn't we extract this from db ? 
Like if a user has 2 years if data, 2 Financial year will be in drop down

Comment (priya-bansal) [reply src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.ts:16]: should not be hard coded I think

File: src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.html (hunk 1 above)
Comment (vgoyal1996) [reply src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.html:10]: Year for now is hardcoded. Later we will see how can we fetch it

File: src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.ts (hunk 2 above)
Comment (vgoyal1996) [reply src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.ts:16]: Yes I know. Will do that later

**Pull request merged**